### PR TITLE
feat: tabs

### PR DESCRIPTION
**Pull request by Vico4**
Issue #1472 : 

- [x] size options 
- [x] spacing
- [x] color
- [x] icon/img/badge size and spacing

---
  

 👀 [Visit Preview](https://feat-tabs--welcome-ui.netlify.app)

### CODE REVIEW

File: packages/Core/theme/tabs.ts [hunk 1]
```diff
@@ -75,5 +81,22 @@ export const getTabs = (theme: WuiTheme): ThemeTabs => {
...
     },
+    size: {
+      sm: {
+        paddingBottom: toRem(9),
+        marginRight: toRem(12),
+      },
+      md: {
+        paddingBottom: toRem(15),
+        marginRight: toRem(24),
+      },
```
Comment (theo-mesnil) [inline packages/Core/theme/tabs.ts:92]: Tou need to get this values from theme

File: packages/Tabs/styles.ts [hunk 2]
```diff
@@ -26,12 +33,11 @@ export const Tab = styled.button`
   display: flex;
   align-items: center;
   flex: none;
-  margin-right: xl;
   padding: lg 0;
   transition: medium;
   text-transform: none;
   cursor: pointer;
-  ${system};
+  gap: 4;
```
Comment (theo-mesnil) [inline packages/Tabs/styles.ts:40]: Get this value from theme

File: packages/Core/theme/tabs.ts (hunk 1 above)
Comment (Vico4) [reply packages/Core/theme/tabs.ts:92]: when value doesn't exist in theme (ie. 9, 15), I go for the closest one or I keep toRem() with requested value ?

Comment (theo-mesnil) [reply packages/Core/theme/tabs.ts:92]: Closest, i think the component is not fully good on design team

**Review by theo-mesnil — APPROVED** [review]

**Pull request merged**